src/Dapr.Actors/Client/ActorProxyOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr.Actors.Client
+{
+    using System.Text.Json;
+
+    /// <summary>
+    /// The class containing customizable optins for how the Actor Proxy works
+    /// </summary>
+    public class ActorProxyOptions
+    {
+        /// <summary>
+        /// The <see cref="JsonSerializerOptions"/> used for actor message serialization
+        /// </summary>
+        public JsonSerializerOptions JsonSerializerOptions { get; set; } = new JsonSerializerOptions();
```

Comment:
Should this be null by default? What will happen here is that this will be non-null by default and so when you pass in the options it will always override the settings in `ActorProxyFactory`.

Reply:
I have removed the `JsonSerializerOptions` from the `ActorProxyFactory` and added a `ActorProxyOptions` property to the `ActorProxyFactory` instead. It seems appropriate that the defaults would be set in the `ActorProxyOptions` and be overridden there if need be.